### ISSUE TITLE
Add ARIA attributes to contact us form

### ISSUE THREAD

**Issue by Avinash905** [akshitagupta15june/PetMe#980]
**Is your feature request related to a problem? Please describe.**
ARIA attributes are crucial for enhancing web application accessibility, benefiting users with disabilities. Currently, the form lacks these attributes, hindering proper interpretation by assistive technologies.

**Describe the solution you'd like**

**Some steps to implement this enhancement-**

- Identify the form elements in your codebase.
- Determine the relevant ARIA attributes to add for each form element.
- Add ARIA roles to the forms and their elements (e.g., "form" role for the main form element).
- Include concise and descriptive ARIA labels using the aria-label attribute.
- Use the aria-describedby attribute to provide additional instructions for the forms.
- Implement ARIA states and properties (e.g., aria-required for required fields).

**Comment by Avinash905** [comment]
Please assign me this issue so that I can work on this issue under GSSOC'23.
